### PR TITLE
Update dependabot and remove buildevents from circle config

### PR DESCRIPTION
**Pull request by paulosman**
Circle changes should allow project to build on PRs from dependabot and others.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,15 +1,9 @@
 version: 2.1
 
 orbs:
-  buildevents: honeycombio/buildevents@0.2.6
   aws-cli: circleci/aws-cli@0.1.13
+  docekr: circleci/docker@1.3.0
```
Comment (robbkidd) [inline .circleci/config.yml:5]: docekr?

```suggestion
  docker: circleci/docker@1.3.0
```

**Review by robbkidd — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (paulosman) [reply .circleci/config.yml]: @robbkidd yours is better

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,137 +22,94 @@ commands:
...
+      - image: circleci/golang:1.13
     steps:
-      - buildevents/with_job_span:
-          steps:
-            - checkout
-            - buildevents/berun:
-                bename: go_test
-                becommand: go test --timeout 10s -v ./...
+      - checkout
+      - run: go test --timeout 10s -v ./...
```
Comment (robbkidd) [inline .circleci/config.yml:35]: If I'm reading this correctly, the `build` job is now responsible for running the tests and there is no `test` job any more, right? I'd expect this job to be named `test`. 😄

Comment (paulosman) [reply .circleci/config.yml:35]: You are making sense.

**Review by robbkidd — APPROVED** [review]
YES

**Pull request merged**